Library/Formula/pinentry-mac.rb

```diff
@@ -13,9 +12,31 @@ class PinentryMac < Formula
     sha1 "4d80f3954ddb16618074d9d978f3a83aedddb8db" => :mountain_lion
   end
 
+  resource "core" do
+    url "https://raw.githubusercontent.com/GPGTools/GPGTools_Core/master/newBuildSystem/prepare-core.sh"
```

Comment:
Solid point. Upstream haven't changed that script in 16 months or so, but I suppose that's no reason to presume they won't change it tomorrow and do something breaking.

> perhaps that should just use git-master instead of a revision, as their script does.

Yeah, I kinda deliberately undid that upstream step. I find pegging stuff to ` git ` master branches is a solid way to have something break suddenly at some point. Using a specific commit point would allow Homebrew more fine-grained bottling and control over the situation, as well as having the benefit of being a slightly stronger checksum mechanism.